/*
Copyright (c) 2024, Alibaba Group;
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/

syntax = "proto3";

import "google/protobuf/empty.proto";

message RequestGroupRequest {
  int32 request_id = 1;
}


message CheckResponse {
  bool is_ok = 1;
  string error_msg = 2;
}


service MockMigrationRequestGroup {
  rpc set_request_group (RequestGroupRequest) returns (google.protobuf.Empty) {}
  rpc check_request_group (RequestGroupRequest) returns (CheckResponse) {}
}
